docs/README-migration.md

```diff
@@ -158,6 +158,11 @@ The standard C headers like stdio.h and stdlib.h are no longer included, you sho
 M_PI is no longer defined in SDL_stdinc.h, you can use the new symbols SDL_PI_D (double) and SDL_PI_F (float) instead.
 
 
+## SDL_surface.h
+
+Removed unused 'depth' field from SDL_CreateRGBSurfaceWithFormat and SDL_CreateRGBSurfaceWithFormatFrom
```

Comment:
This should be added to WhatsNew.txt as well, which documents all API changes from the previous version.